ddr/lib/ddr-scanner/dwarf/DwarfParser.cpp

```diff
@@ -1,19 +1,22 @@
 /*******************************************************************************
+ * Copyright (c) 2015, 2017 IBM Corp. and others
  *
- * (c) Copyright IBM Corp. 2015, 2017
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at http://eclipse.org/legal/epl-2.0
+ * or the Apache License, Version 2.0 which accompanies this distribution
+ * and is available at https://www.apache.org/licenses/LICENSE-2.0.
  *
- *  This program and the accompanying materials are made available
- *  under the terms of the Eclipse Public License v1.0 and
- *  Apache License v2.0 which accompanies this distribution.
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the
+ * Eclipse Public License, v. 2.0 are satisfied: GNU General Public License,
+ * version 2 with the GNU Classpath Exception [1] and GNU General Public
+ * License, version 2 with the OpenJDK Assembly Exception [2].
  *
- *      The Eclipse Public License is available at
- *      http://www.eclipse.org/legal/epl-v10.html
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
  *
- *      The Apache License v2.0 is available at
- *      http://www.opensource.org/licenses/apache2.0.php
- *
- * Contributors:
- *    Multiple authors (IBM Corp.) - initial implementation and documentation
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0

```

Comment:
there's a busted comment at line 157 in this file that happens to reference the epl-v10 file name. It was there before this PR and the comment has nothing to do with our licensing, just pointing it.

Reply:
Sadly this was in the original version of the file.  I am not sure what it is supposed to be.  I will reach out to the developer to see if they have any input but at this point I am going to ignore if for this PR

Reply:
@jason-hall Can you weigh in on what that comment should say? Is the reference to epl-v10 significant?